specifications/api/swagger.yaml

```diff
@@ -631,6 +631,11 @@ responsesErr404: &responsesErr404
     description: Not Found
     schema: *responsesErr
 
+responsesErr406: &responsesErr406
+  406:
+    description: Not Acceptable
+    schema: *responsesErr
```

Comment:
There are actually quite a few endpoints that can return a 406 ... Shall we update them all in this PR?

Reply:
I was thinking about updating spec along with the testing of particular endpoints, making sure what sort of errors they can produce etc.... but I suppose they will all produce 406, so why not.